src/main/java/core/basesyntax/RandoM.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandoM extends Enum {
```

Comment:
The class name 'RandoM' is unconventional and may cause confusion. Class names should be nouns and written in UpperCamelCase. Also, extending 'Enum' is not appropriate here, as 'Enum' is a special class in Java used for defining enums and should not be used as a base class for other classes.